make/cpp2asm_support.cmake

```diff
@@ -132,7 +132,14 @@ endif ()
 ##################################################
 # Preprocessor location and flags
 
-if (UNIX)
+if (MSVC)
```

Comment:
OK, good, this predefined cmake variable `MSVC` will work.  Maybe better to do `if (NOT MSVC)` to avoid code churn here from the original code (i.e., avoid having to move code around).

Reply:
It seems to me `if (MSVC)` is easier to understand, and much simpler for adding new conditions to `else`.